Packs/IntSight/Integrations/Rapid7ThreatCommand/Rapid7ThreatCommand.py

```diff
@@ -563,11 +565,10 @@ def alert_fetch_parser(
                 ],
                 "is_closed": dict_safe_get(obj, ["Closed", "IsClosed"]),
                 "sub_type": dict_safe_get(obj, ["Details", "SubType"]),
-                "attachments": attachments,
             }
         )
 
-    def parse_incident(self, alert: dict, attachments: List[dict[str, Any]]) -> dict:
+    def parse_incident(self, alert: dict) -> dict:
```

Comment:
just want to verify with you,
the incidents are now fetched without attachments and csv files from the integration, instead they will added later with `Retrieve Alert Attachments - Rapid7 ThreatCommand` playbook inside the incident?

Reply:
Yes, correct.